app/views/channels.html

```diff
@@ -49,6 +49,7 @@
                   <td>
 
                     <div ng-if="channel.status!='deleted'">
+                      <button class="btn btn-info btn-xs" ng-click="duplicateChannel(channel)"><i class="glyphicon glyphicon-file"></i></button>
```

Comment:
I think we need some tooltip text for this as it is not clear what it does from just looking at it.

While you are at it, perhaps adding tooltips to the two other buttons would be a good idea.